packages/data-models/src/language.js

```diff
@@ -19,18 +20,26 @@ export default class Language {
   /**
    * Creates an instance of a language class.
    *
-   * @param {Lang} code - A constant that specifies a language.
+   * @param {Lang | string} code - A constant that specifies a language in an ISO 639-3 format.
```

Comment:
Since we are now using the Language class to encapsulate languages other than those that initiated their creation (e.g. for a Definition, we have the language of the lemma of the definition and the language of the text of the definition) we should explicitly support ISO 639-2 codes as well as ISO 639-3. For Alpheios-supported languages we normalize to ISO 639-3 but for other languages we should support either, and we could validate against the standard list of 639-3 and 639-2 codes.

Reply:
It might be a work with many nuances. I've created an issue https://github.com/alpheios-project/alpheios-core/issues/597 to track that.